WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -18,6 +23,29 @@ final class WeatherViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        setUpView()
+        bindOutput()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        updateConstraintForCurrentOrientation()
+    }
+
+    private func bindOutput() {
+        viewModel.output.temperature.emit(with: self) { owner, temperatureValue in
+            owner.temperatureLabel.text = temperatureValue
+        }.disposed(by: disposeBag)
```

Comment:
About convention, I believe we expect to see like this

```suggestion
        viewModel.output.temperature
            .emit(with: self) { owner, temperatureValue in
                owner.temperatureLabel.text = temperatureValue
            }
            .disposed(by: disposeBag)
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/a14f6fdfc45ee914dd976766a33e952f0c779916)